app/comparator.py

```
@@ -0,0 +1,134 @@
+import subprocess
+import os
+import json
+from datetime import datetime
+from typing import Optional, Tuple
+from .models import ComparisonResult
+from deepdiff import DeepDiff
+
+class OpenAPIComparator:
+    def __init__(self, script_path: str = "./compare_openapi.sh"):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)